400-claims.md

```diff
@@ -0,0 +1,197 @@
+
+```console
+$ brew info cscope
+cscope: stable 15.8b (bottled)
+Tool for browsing source code
+https://cscope.sourceforge.io/
+/usr/local/Cellar/cscope/15.8b (10 files, 714.2KB) *
+  Poured from bottle on 2017-05-15 at 09:24:58
+From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/cscope.rb
+```
+
+The CNAB Claims specification does not define where or how records are stored, nor how these records MAY be used by CNAB tooling. However, this specification describes how a CNAB-based system MUST emit the record to an invocation image, and provides some guidance on maintaining integrity of the system.
+
+This is done so that implementors can standardize on a way of relating a release claim (the record of a release) to release operations like `install`, `upgrade`, or `uninstall`. This, in turn, is necessary if CNAB bundles are expected to be executable by different implementations.
+
+### Anatomy of a Claim
+
+While implementors are not REQUIRED to implement claims, this is that standard format for claims-supporting systems.
+
+The CNAB claim is defined as a JSON document. The specification currently does not require that claims be formatted as Canonical JSON.
+
+```json
+{
+    "name": "technosophos.hellohelm",
+    "revision": "01CP6XM0KVB9V1BQDZ9NK8VP29",
+    "created": "2018-08-30T20:39:55.549002887-06:00",
+    "modified": "2018-08-30T20:39:59.611068556-06:00",
+    "bundle": {
+      "schemaVersion": "v1.0.0-WD",
+      "name": "technosophos.hellohelm",
+      "version": "0.1.0",
+      "invocationImages": [
+        {
+          "imageType": "docker",
+          "image": "technosophos/demo2:0.2.0"
+        }
+      ],
+      "images": {},
+      "parameters": {},
+      "credentials": {}
+    },
+    "result": {
+      "message": "",
+      "action": "install",
+      "status": "success"
+    },
+    "parameters": {}
+  }
+```
+
+- name: The name of the _installation_. This can be automatically generated, though humans may need to interact with it. It MUST be unique within the installation environment, though that constraint MUST be imposed externally. Elsewhere, this field is referenced as the _installation name_.
+- revision: An [ULID](https://github.com/ulid/spec) that MUST change each time the release is modified.
+- bundle: The bundle, as defined in [the Bundle Definition](101-bundle-json.md)
+- created: A timestamp indicating when this release claim was first created. This MUST not be changed after initial creation.
+- updated: A timestamp indicating the last time this release claim was modified
+- result: The outcome of the bundle's last action (e.g. if action is install, this indicates the outcome of the installation.). It is an object with the following fields:
+  - message: A human-readable string that communicates the outcome. Error messages MAY be included in `failure` conditions.
+  - action: Indicates the action that the current bundle is in. Valid actions are:
```

Comment:
Should we revisit this list of valid actions (or re-word)?  Thinking of the well-known custom actions added in https://github.com/deislabs/cnab-spec/pull/102.

(If appropriate to amend, but list not introduced/changed by this PR, I would understand relegating as follow-up.)

Reply:
Good catch! I totally missed that part, and several of the action names weren't even correct.